dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -664,6 +673,7 @@ public Workflow updateWorkflowPath(@ApiParam(hidden = true) @Auth User user,
     public Workflow getPublishedWorkflow(@ApiParam(value = "Workflow ID", required = true) @PathParam("workflowId") Long workflowId) {
         Workflow workflow = workflowDAO.findPublishedById(workflowId);
         checkEntry(workflow);
+        workflow.getVersions().forEach(workflowVersion -> Hibernate.initialize(workflowVersion.getValidations()));
```

Comment:
Same with this and the two below, I think a query parameter  (default to current content) might be least disruptive